src/store/getters/state-getters.ts

```diff
@@ -119,7 +118,7 @@ export const isShowBackBtn = (state: any): boolean => {
  * Whether Review and Confirm button should be displayed.
  */
 export const isShowReviewConfirmBtn = (state: any, getters: any): boolean => {
-  return (!!state.stateModel.nameRequest.entityType && state.stateModel.currentStep < getters.getMaxStep)
+  return (!!state.stateModel.entityType && state.stateModel.currentStep < getters.getMaxStep)
```

Comment:
Any idea why we need entity type to be defined to show this button? Seems to me it's always defined (and if so then we can remove this check).
Maybe ask Cam or Lekshmi.

Reply:
Good point, do we ever not have a defined entityType? This could have been something implemented very early. Might want to look into removing the entityType boolean.